### DISCUSSION TITLE
List of participants

### DISCUSSION THREAD

**Discussion by wodbow** [lukevella/rallly#127] [Ideas]
Hi, first of all thank for this tool, is the best of it's kind I've seen so far (at least for my purposes).
Really easy and quick for everyone involved.

I've noticed that in this new version I can't see a full list of participants who can/cannot attend in a simple view, I just see his/her name initial. If I want to check who are the people attending, I have to check one by one instead of having them all in the same place.

Is it me, or is it something that's not available anymore?

Thank you!

**Comment by lukevella** [comment]
Hi, if you're on your phone the compact view does not show you the list of participants you're right but for now you could tilt your phone horizontally to get the tabular view. Another option is to export your results to CSV and open them in a spreadsheet on your phone but I understand this is not ideal. I will make it possible to get a full list of names for each option on mobile soon.

**Comment by wodbow** [comment]
Thanks for the answer, and for the tip about tilting the phone, I hadn't
thought about that.

And again, thanks for the hard work!

**Comment by pagenoare** [comment]
On horizontal mobile view, the scrolling is a bit off and list of participants is hiding very quickly... lmk if you need video to see what I mean.

**Reply by lukevella** [reply]
Hi, I think I understand what you mean. The recent update means that the poll header is using more vertical space so on smaller screens the list of participants might not even be visible. I will try and make the header a bit more compact in the future but I do also plan to update the mobile version of the poll soon so that it won't be necessary to go horizontal to list participants.
